runners/spark/3/src/main/java/org/apache/beam/runners/spark/structuredstreaming/translation/SparkSessionFactory.java

```diff
@@ -283,5 +283,13 @@ public void registerClasses(Kryo kryo) {
       kryo.register(TupleTag.class);
       kryo.register(TupleTagList.class);
     }
+
+    private void tryToRegister(Kryo kryo, String className) {
+      try {
+        kryo.register(Class.forName(className));
+      } catch (ClassNotFoundException e) {
+        LOG.warn("Class {}} was not found on classpath", className);
```

Comment:
Nit, just log these as info? It's kind of expected that these are missing.

Reply:
Makes sense, done